### PR TITLE
add job to update galaxy-ng and deps nightly

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/pulpcore_packaging_nightly.groovy [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            }
+        }
+
+        stage('Nightly Build Packages') {
+            steps {
+                setup_obal()
+
+                script{
+
+                    //def response = httpRequest "${source_server}/index.json"
```
Comment (ehelms) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/pulpcore_packaging_nightly.groovy:35]: Drop this line?

File: puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging/pulpcore-packaging-rpm-nightly.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+- job:
+    name: pulpcore-packaging-rpm-nightly
+    project-type: pipeline
+    concurrent: true
```
Comment (ekohl) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging/pulpcore-packaging-rpm-nightly.yaml:5]: Does it make sense to run these concurrently? Won't you end up with duplicate builds?

Comment (evgeni) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging/pulpcore-packaging-rpm-nightly.yaml:5]: the pipeline file has concurrent no, and that seems to win anyways :see_no_evil:

File: puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/pulpcore_packaging_nightly.groovy [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        }
+
+        stage('Nightly Build Packages') {
+            steps {
+                setup_obal()
+
+                script{
+
+                    def response = sh returnStdout: true, script: "curl --silent ${source_server}/index.json"
+                    def index = readJSON text: response
```
Comment (ekohl) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/pulpcore_packaging_nightly.groovy:36]: Have you thought about error handling?

File: puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/pulpcore_packaging_nightly.groovy [hunk 4]
```diff
@@ -0,0 +1,69 @@
+def packaging_branch = 'rpm/3.7'
+def source_server = 'http://ah.testing.ansible.com/nightlies/latest/'
```
Comment (ekohl) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/pulpcore_packaging_nightly.groovy:2]: I'm not very comfortable with pulling instructions over plain text HTTP. Is it available over something secure?

Comment (evgeni) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/pulpcore_packaging_nightly.groovy:2]: Not today, but we can ask Yanis.

File: puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/pulpcore_packaging_nightly.groovy (hunk 3 above)
Comment (evgeni) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/pulpcore_packaging_nightly.groovy:36]: Not so far. Do you mean handling bad JSON or HTTP errors?

Comment (ekohl) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/pulpcore_packaging_nightly.groovy:36]: Yes

File: puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/pulpcore_packaging_nightly.groovy (hunk 4 above)
Comment (ekohl) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/pulpcore_packaging_nightly.groovy:2]: Please do.

**Comment by ehelms** [comment]
We have moved the Jenkins job definitions over to https://github.com/theforeman/jenkins-jobs -- please re-open your PR against this new repository.

**Pull request closed without merging**